### PR TITLE
HelloWorld-Microverse

### PR DESCRIPTION
**Pull request by M-Anwar-Hussaini**
Hi dear Code Reviewer.

I tried my best to fix the issues you mentioned in the first PR. I hope they are all fixed.
If there are some some problems please do mention me so that I consider in the future projects.

Thanks a lot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,31 +238,17 @@ If you like this project...
...
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (usorfaitheloho) [inline README.md]: I'd kindly suggest you remove all comments, and descriptive lines in the README file to make your code legible and easy to understand 😄 . You should also edit the contents of all required sections with the right details of your project, you did well with the `authors` section so please implement the remaining sections 👍

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @M-Anwar-Hussaini , 

>You are almost there!
Great effort so far, but unfortunately there are some issues that you need to review in order to get your project approved, but you got this!


Alt Text
>


## Highlights
- Correct GitHub flow was used :100: 
- Linter checks are passing :100: 
<br/>
<br/>
<br/>

## Required Changes ♻️
- [x] Please add a descriptive Pull Request Title and summary using the project details to give a better understanding of the changes you introduced in this milestone.
- [x]  You did great using the right Microverse README template, please always remember to only use the sections that are relevant to your project and fill all sections with the appropriate content relative to the work you did for the project.

- [x] As part of the requirements, please create a `test. md` file in the root directory of your repository. You did well in adding it to the `.gitignore` file so go ahead and create it in your project

test md


_Check the comments below to see technical issues found during this review._

## Optional suggestions


- [x] Please ensure you use descriptive branch names that summarize the features you implemented in a repository, and for each commit message  also use good and imperative commit messages giving reasons for the actions completed in the description

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
<br/>


You can also try reaching me on Slack, looking for me as `@Javier Hernandez`

Cheers and Happy coding!👏👏👏
<br/>
<br/>
<br/>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by M-Anwar-Hussaini** [comment]
Thanks.
I will figure it out.

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [ ] Please kindly work on Your readme file as it is currently not looking very professional. It is great you used the [Microverse readme template](https://github.com/microverseinc/readme-template) but you didn't follow it to the latter. Every section of the template is required to appear on your read me and you are to customise the template with your github repo details as well as details needed for better description of the project. :+1: 
 - [ ] Your readme is currently missing the following sections `Getting started, setup, prerequisites, install, usuage, run tests, deployment, future features, contributing, show your support` please kindly add them before opening a new pull request. :+1:

File: README.md [hunk 2]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
```
Comment (Efiamotu-1) [inline README.md]: please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off +1

File: README.md [hunk 3]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [HelloWorld-Microverse] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```
Comment (Efiamotu-1) [inline README.md]: please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1:

File: README.md [hunk 4]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
```
Comment (Efiamotu-1) [inline README.md]: - [ ] comments are unused code which do not make your project look professional, do well to remove them :+1:

File: README.md [hunk 5]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Kindly ensure that you are using the correct technologies you used in building this project as it is more descriptive and anyone even recruiters can use and test your projects. :+1:

File: README.md [hunk 6]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+**[HelloWorld-Microverse]** is a the first project I did during Microverse journey in order to practise Linters.
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
```
Comment (Efiamotu-1) [inline README.md]: please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1:

File: README.md [hunk 7]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
```
Comment (Efiamotu-1) [inline README.md]: - [ ] You are required to list at least 2-3 features which you implemented on this branch please try to do this as it is professional and shows how much you progress with your work with every given branch :+1:

File: README.md [hunk 8]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
```
Comment (Efiamotu-1) [inline README.md]: please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1:

File: README.md [hunk 9]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
-- [Live Demo Link](https://google.com)
```
Comment (Efiamotu-1) [inline README.md:114]: - [ ] Kindly notify users when the live demo is not ready or if it won't be implemented. This prevents your Readme from having broken links :+1:

File: README.md [hunk 10]
```diff
@@ -60,229 +60,53 @@ After you're finished please remove all the comments and instructions!
...
-```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
 > Mention all of the collaborators of this project.
```
Comment (Efiamotu-1) [inline README.md]: - [ ] please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1:

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] It is also important to use good PR title and summary when making a pull request attached below is an image of how your pr title and summary for this project should look like please make sure to make the correct changes before resubmitting for a new pull request. :+1: 
- [ ] Make sure to also name your branches descriptively tailored to the changes you would be making for example `add-homepage`, `mobile-menu-feature` etc
good-pr

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @M-Anwar-Hussaini , 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- Good job implementing some of the requirements of the first reviewer :+1: 
- ✔️ No linter Errors
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (M-Anwar-Hussaini) [reply README.md]: Hi @Efiamotu-1 
Could you please explain this part with more details.

Thanks.

Comment (Efiamotu-1) [reply README.md]: @M-Anwar-Hussaini the hello microverse project you submitted was built using html and css but what you have on your readme file here is showing reactJs, expressJs which you didn't use at all. so the change is basically for you to use the correct technologies which are html and css in your readme under the built with section. did you find the explanation helpful?

**Review by taldr27 — APPROVED** [review]
## Hi @M-Anwar-Hussaini , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.

## Optional suggestions 💭

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] **[OPTIONAL]** You did a great job on this project, I just have a small suggestion to make your future pull requests even better, always try to write your pull request summary describing all the changes you made, I will tag you on the previous reviewer comment. 💯

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: index.html
Comment (taldr27) [reply index.html:1]: - [ ] @M-Anwar-Hussaini Nice work making all the requested changes from previous reviewers, I just have one suggestion to make your pull request more professional in this and your future projects, always try to write your pull request summary like in the image provided by @Efiamotu-1. 💯

**Pull request merged**